README.md

```diff
@@ -9,10 +9,42 @@ mkdir -p code/var/cache/dev code/var/log
 chmod -R 777 code/var
 ```
 
+Build the environnement
+```
+docker-compose build
+docker-compose run php composer install
+```
+
 Run the web server:
 
 ```sh
 docker-compose up
 ```
 
 Then the website is available at `http://localhost:8080`.
+
+## Deploying on Scalingo
```

Comment:
I would move this section before "Running locally" as it is probably the most interesting for our customers.